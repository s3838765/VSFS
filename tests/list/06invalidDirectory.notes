NOTES V1.0
=validDir/
=invalidDir
