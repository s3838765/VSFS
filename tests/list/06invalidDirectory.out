The directory invalidDir is not correctly formatted (must end with a "/"). Terminating program.
